ConwaysGameOfLife.Tests/GameControllerShould.cs

```diff
@@ -18,128 +18,150 @@ public GameControllerShould()
-                var actualOutputLine2 = _output.GetWriteLine(i + 1);
-                Assert.Equal(invalidOutputMessageLine2, actualOutputLine2);
-            }
+            _input.GetReadLine("q");
 
-            Assert.Equal(validInput, result);
-        }
-
-        [Theory]
-        [InlineData("0,0", "3,3")]
-        [InlineData("0,0", "4,3")]
-        [InlineData("2,1", "10,5")]
-        [InlineData("2,1", "4,6")]
-        public void ReturnLocationString_GivenValidInput(string input, string universeInput)
-        {
-            _input.GetReadLine(universeInput);
-            _controller.CreateInitialUniverse();
-            _input.GetReadLine(input);
-            var expected = input;
-
-            var result = _controller.CreateValidLocationString();
+            _controller.Run();
+            var actual = _output.GetLastWriteLine();
 
-            Assert.Equal(expected, result);
+            Assert.Equal(expected, actual);
         }
 
         [Theory]
-        [InlineData(new string[] { "0,3" }, "2,2", "3,3")]
-        [InlineData(new string[] { "3,0" }, "0,0", "3,3")]
-        [InlineData(new string[] { ",,", ";;" }, "2,2", "10,10")] // failing
-        [InlineData(new string[] { "3,3,", "3,4;" }, "3,2", "4,4")] // failing
-        [InlineData(new string[] { "3,3,3" }, "3,2", "5,4")]
-        [InlineData(new string[] { "a,b", "0,a", "o,9" }, "1,1", "2,2")]
-        [InlineData(new string[] { "-1,4", "a,b", "0,a", "o,9" }, "2,0", "3,3")] // failing
-        [InlineData(new string[] { "4,-1" }, "2,2", "5,5")]
-        [InlineData(new string[] { "4 4" }, "2,1", "3,3")]
-        public void ReturnLocationString_GivenInvalidInputsFollowedByValidInput(IEnumerable<string> invalidInputs, string validInput, string universeInput)
-        {
-            _input.GetReadLine(universeInput);
-            _controller.CreateInitialUniverse();
-            _input.GetReadLine(invalidInputs);
-            _input.GetReadLine(validInput);
-            var invalidOutputMessageLine1 = "Invalid input. Please try again.";
-            var invalidOutputMessageLine2 = "Please enter the x, y coordinates for one live cell (e.g. '0,1').";
-            var count = invalidInputs.Count();
-
-            var result = _controller.CreateValidLocationString();
-
-            for (var i = 1; i <= count; i += 2)
-            {
-                var actualOutput = _output.GetWriteLine(i);
-                Assert.Equal(invalidOutputMessageLine1, actualOutput);
-                var actualOutputLine2 = _output.GetWriteLine(i + 1);
-                Assert.Equal(invalidOutputMessageLine2, actualOutputLine2);
-            }
-
-            Assert.Equal(validInput, result);
-        }
-
-        [Fact]
-        public void SetLiveCellLocation()
+        [InlineData("3,3", "1,2", "💀💀💀\n💀💀💟\n💀💀💀\n")] // failing - displays live cell @ location 1,2
+        [InlineData("3,3", "2,0", "💀💀💀\n💀💀💀\n💟💀💀\n")] // failing - displays live cell @ location 0,2

```

Comment:
```suggestion
        [InlineData("3,3", "1,2", "💀💀💀\n💀💀💀\n💀💟💀\n")]
        [InlineData("3,3", "2,0", "💀💀💟\n💀💀💀\n💀💀💀\n")]
```
it's the expected value that seems to be off


x,y so 1,2 would be 
💀💀💀
💀💀💀
💀💟💀

and 2,0 would be
💀💀💟
💀💀💀
💀💀💀

Reply:
What do I do so that I don't keep making this same mistake?